### DISCUSSION TITLE
Issue with big messages

### DISCUSSION THREAD

**Discussion by davidbrochart** [frankie567/httpx-ws#22] [Q&A]
I'm not sure the issue is in `httpx-ws`, but I have a WebSocket server which receives truncated messages from `httpx-ws`. I've tried to increase [max_message_size_bytes](https://frankie567.github.io/httpx-ws/reference/httpx_ws/#httpx_ws.aconnect_ws) to an arbitrary high value, but it doesn't seem to help.
Sorry I don't have more details, but maybe there is something obvious off the top of your head?

**Comment by frankie567** [comment]
Hmmm 🤔 Does your server run with Uvicorn? If so, have you tried to tinker with the [`ws-max-size` setting](https://www.uvicorn.org/settings/#implementation)?

**Reply by davidbrochart** [reply]
Found it, it was completely unrelated, sorry for the noise!
